benchmarks/benchmarks-game/mandelbrot.lisp

```diff
@@ -0,0 +1,179 @@
+(defpackage #:coalton/benchmarks/benchmarks-game/mandelbrot
```

Comment:
we should write a comment on what this code is benchmarking. i don't mean documenting what mandelbrot is, but rather "this code benchmarks highly polymorphic complex numerics, and high order functions. the code is written in a relatively idiomatic functional style." or so